### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by smgladkovskiy**
- [x] Refactor logger to be as a logrus wrapper only
- [x] DB and Mux loggers are dependant objects that initiates separately
- [x] Make Sentry hook work with application stacktrace, not only with logger package stacktrace
- [x] Add readme
- [x] Add licence
- [x] Add tests
- [x] Add CircleCI to test it
- [x] Add badges 😍

### CODE REVIEW

File: gin/muxLogger.go [hunk 1]
```diff
@@ -7,8 +7,16 @@ import (
...
 
+type MuxLogger struct {
+	l *logrus.Logger
+}
+
+func NewLogger(logger *logrus.Logger) *MuxLogger {
+	return &MuxLogger{l: logger}
+}
+
 // MuxLogger Логирование работы веб-сервера
```
Comment (roquie) [inline gin/muxLogger.go]: eng

File: gorm/dbLogger.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+type DBLogger struct {
+	l *logrus.Logger
+}
+
+func NewLogger(logger *logrus.Logger) *DBLogger {
+	return &DBLogger{l: logger}
+}
+
+// Лоирование работы БД
```
Comment (roquie) [inline gorm/dbLogger.go]: eng

File: config.go [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package logs
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultSTAGE = "unknown"
```
Comment (roquie) [inline config.go]: Any our microservice must have "defaults" stage as default.

File: config.go [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+type StackTraceConfig struct {
+	Enable  bool
+	Context int `yaml:"context"`
+}
+
+type SentryConfig struct {
+	Enable          bool             `yaml:"enable"`
+	Stage           string           `yaml:"stage"`
+	MinlLogLevel    string           `yaml:"min_log_level"`
```
Comment (roquie) [inline config.go]: maybe use just "level" key for the most compatibility with old services?

File: config.go [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+	DSN             string           `yaml:"dsn"`
+	ResponseTimeout time.Duration    `yaml:"response_timeout"`
+	StackTrace      StackTraceConfig `yaml:"stacktrace"`
+}
+
+type Config struct {
+	Level  string        `yaml:"level"`
```
Comment (roquie) [inline config.go]: and what?

File: config.go (hunk 4 above)
Comment (smgladkovskiy) [reply config.go]: 1. log level info.
2. sentry min log level error.

StdOut logs will fires from 1.
Sentry will logs from 2.

File: config.go (hunk 5 above)
Comment (smgladkovskiy) [reply config.go]: see previous comment answer

**Pull request merged**